docs/proposals/multiple-schedulers.md

```diff
@@ -0,0 +1,139 @@
+<!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
+
+<!-- BEGIN STRIP_FOR_RELEASE -->
+
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+<img src="http://kubernetes.io/img/warning.png" alt="WARNING"
+     width="25" height="25">
+
+<h2>PLEASE NOTE: This document applies to the HEAD of the source tree</h2>
+
+If you are using a released version of Kubernetes, you should
+refer to the docs that go with that version.
+
+<strong>
+The latest 1.0.x release of this document can be found
+[here](http://releases.k8s.io/release-1.0/docs/proposals/multiple-schedulers.md).
+
+Documentation for other releases can be found at
+[releases.k8s.io](http://releases.k8s.io).
+</strong>
+--
+
+<!-- END STRIP_FOR_RELEASE -->
+
+<!-- END MUNGE: UNVERSIONED_WARNING -->
+
+# WIP: Multi-Scheduler in Kubernetes
+
+**Author**: Haiyang DING(@HaiyangDING)
+
+**Status**: Design & Implementation in progress.
+
+## Motivation
+
+In current Kubernetes design, there is only one default scheduler in a Kubernetes cluster.
+However in some scenarios, users may want to launch user-provided schedulers to enable some
+customized scheduling behavior. This is one of the reasons we should consider multiple schedulers
+in Kubernetes. As previously discussed in [#11793](https://github.com/kubernetes/kubernetes/issues/11793),
```

Comment:
Instead of pointing to issues, we can improve this by summarizing the key point of motivation:
- Allow multiple scheduler running simultaneously. Let user add/plugin custom scheduler.
- Flexible scheduling policy.
- Scalability
- ...

We can put related issues into Reference section.